test/unit/AccountPageModal.test.js

```diff
@@ -0,0 +1,245 @@
+      sandbox.stub(global, 'setTimeout').callsFake((fn, timeout) => {
+        callRecord.push({ setTimeout: timeout });
+        fn();
+      });
+      sandbox.stub(React, 'useState').callsFake((arg) => {
+        global.update = arg;
+        return [
+          global.update,
+          (setUpdateArg) => {
+            callRecord.push({ setUpdate: setUpdateArg });
+          },
+        ];
+      });
+
+      component = mount(
+        <AccountPageModal
+          stayLoggedIn={() => {}}
+          baseUrl="fakeBaseUrl"
+        />,
+      );
+    });
+
+    it('should set a timeout to update in 1 second', () => {
+      console.log('callRecord: ', callRecord);
+      expect(callRecord.some(call => call.setTimeout === 1000)).to.equal(true);
+      expect(callRecord.some(call => call.setUpdate === true)).to.equal(true);
+    });
+
+    it('should not trigger a logout', () => {
+      expect(callRecord.some(call => call.replace)).to.equal(false);
+    });
+
+    it('should render null', () => {
+      console.log('component: ', component);
+      expect(component.instance()).to.equal(null);
+    });
+
+    after(() => {
+      sandbox.restore();
+      clock.restore();
+    });
+  });
+
+  describe('when between 0 and 3 minutes remain', () => {
+    const callRecord = [];
+    let component;
+    let sandbox;
+
+    before(() => {
+      sandbox = sinon.sandbox.create();
+
+      window.location = { replace: () => {} };
+      global.document.cookie = `accountPageExp=${((2 * 60) + 3) * 1000}`;
+      sandbox.stub(window.location, 'replace').callsFake(() => { callRecord.push({ replace: true }); });
+      sandbox.stub(global, 'setTimeout').callsFake((fn, timeout) => {
+        callRecord.push({ setTimeout: timeout });
+        fn();
+      });
+      sandbox.stub(Date.prototype, 'getTime').callsFake(function () {
+        return this.toString().includes('123000') ? 123000 : 0;
+      });
+      sandbox.stub(React, 'useState').callsFake((arg) => {
+        global.update = arg;
+        return [
+          global.update,
+          (setUpdateArg) => {
+            callRecord.push({ setUpdate: setUpdateArg });
+          },
+        ];
+      });
+
+      component = mount(
+        <AccountPageModal
+          stayLoggedIn={() => {}}
+          baseUrl="fakeBaseUrl"
+        />,
+      );
+    });
+
+    it('should set a timeout to update in 1 second', () => {
+      console.log('callRecord: ', callRecord);
+      expect(callRecord.some(call => call.setTimeout === 1000)).to.equal(true);
+      expect(callRecord.some(call => call.setUpdate === true)).to.equal(true);
+    });
+
+    it('should not trigger a logout', () => {
+      expect(callRecord.some(call => call.replace)).to.equal(false);
+    });
+
+    it('should say \'Your session is about to time out\'', () => {
+      expect(component.find(Card).text()).to.include('Your session is about to time out');
```

Comment:
You should be able to use the string `'Card'` and not import the component

Reply:
Fixed. Also same for `Button`